package.json

```diff
@@ -1,10 +1,18 @@
 {
   "name": "@politico/vue-accessible-selects",
-  "version": "0.0.1",
+  "version": "0.0.2-alpha.5",
   "description": "Select & Multi Select implementations for Vue, focused especially on implementing accessibility best practices",
-  "main": "index.js",
+  "files": [
+    "dist"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
   "scripts": {
-    "test": "echo \"write some tests\""
+    "test": "echo \"write some tests\"",
+    "build": "NODE_ENV=production rollup --config rollup.config.js && rm -rf dist/styles && cp -r src/styles dist/styles",
+    "prepublishOnly": "npm run build",
```

Comment:
Did you add this for use in a script or other convenience?

Reply:
Convenience only, to run when I feel like it. It was just to test whether the exported files are indeed tree-shakeable. We could optionally add it as part of `prepublishOnly`? I think it would "correctly" fail if the output is not tree-shakeable